test/screenshot/spec/mdc-menu/fixture.js

```diff
@@ -29,7 +31,7 @@ window.mdc.testFixture.fontsLoaded.then(() => {
   menu.open = true;
 
   buttonEl.addEventListener('click', () => {
-    menu.setDefaultFocusItemIndex(mdc.menu.MDCMenuFoundation.numbers.FOCUS_ROOT_INDEX);
+    menu.setDefaultFocusState(DefaultFocusState.LIST_ROOT);
```

Comment:
Why do you need this if it is the default?

Reply:
The previous `handleMenuSurfaceOpened` calls in `keydown` callback below overrides the default focus state. It is required every time before menu is opened.